crates/bindings-macro/src/lib.rs

```diff
@@ -1029,7 +1029,19 @@ fn table_impl(mut args: TableArgs, mut item: MutItem<syn::DeriveInput>) -> syn::
     let unique_col_ids = unique_columns.iter().map(|col| col.index);
     let primary_col_id = primary_key_column.iter().map(|col| col.index);
     let sequence_col_ids = sequenced_columns.iter().map(|col| col.index);
-    let scheduled_reducer_ident = args.scheduled.iter();
+
+    let schedule = args
+        .scheduled
+        .as_ref()
+        .map(|reducer| {
+            // scheduled_at was inserted as the last field
```

Comment:
I was wondering if we wanted to have the users declare the column as well so they can see what is going on + sometimes get better intellisense, but that could be done as a follow-up PR if we decide to do it.

Reply:
Yes that's the plan!